sdk/iot/hub/src/az_iot_hub_client_sas.c

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: MIT
 
 #include <az_iot_hub_client.h>
+#include <az_iot_core_internal.h>
```

Comment:
az_iot_common_internal.h

Reply:
Yes, I just posted an update to this PR with the renaming. Thanks.